ext/serializer.c

```diff
@@ -319,11 +312,13 @@ static int dd_fatal_error_to_meta(zend_array *meta, dd_error_info error) {
     return error.type && error.msg ? SUCCESS : FAILURE;
 }
 
-static zend_result dd_add_meta_array(void *context, ddtrace_string key, ddtrace_string value) {
-    zval *meta = context, tmp = ddtrace_zval_stringl(value.ptr, value.len);
+static zend_result dd_add_meta_array(ddog_SpanBytes *span, ddtrace_string key, ddtrace_string value) {
+    ddog_ZendString *key_zstr = (ddog_ZendString *)zend_string_init(key.ptr, key.len, 0);
```

Comment:
May I suggest to use
```
[export.rename]
"ZendString" = "_zend_string"
```
in cbindgen.toml, to avoid casting stuff around all the time?